README.md

```diff
@@ -1,29 +1,88 @@
 
+### Description
 The definitive Anti X-Ray plugin for CraftBukkit
+This plugin is used to counter X-RAY client mods, texture packs, chest radar, and other similar exploits
+It modifies data that are sent to clients to hide blocks of your choice, such as ore, chests, dungeons, etc.
+It does not manipulate blocks in the world file, thus is safe to use.
+ProximityHider is a feature that hides chests that are far from players.
+
+### Features:
+- Advanced algorithm that hides ore, chest and and anything you specify in the configuration
+- Spout compatible, but optional
+- No modifications to CraftBukkit.jar is needed
+- Customize the blocks you want to hide
+- HIDES DUNGEONS and other blocks that are in the dark
+- Different hiding mode, or scrambling.
+- Extensive configuration. Change updating methods depending on your bandwidth and processing speed.
+- Hide hidden chests and furnaces until a player is close to it.
+- Hide hidden chests and furnaces until a player can see it.
+
+### Links
+- **[Discord](https://chat.wuffy.eu)**
+- **[Orebfuscator](https://www.spigotmc.org/resources/orebfuscator.22818/)**
+- **[ProtocolLib](https://www.spigotmc.org/resources/protocollib.1997)**
 
-### Changelog:
-- Updated to support 1.11 - 1.11.2
-- Updated to support 1.10 (Thanks Asgorioth)
-- Updated to support 1.9.4
-- Updated to support 1.9-1.9.2 (https://www.devotedmc.com)
+## IMPORTANT
 
 ### Requirements:
 - Java 1.6 / 1.7 / 1.8
-- Spigot/PaperSpigot and any other fork of CraftBukkit! (1.9 through 1.10)
+- Spigot/PaperSpigot and any other fork of CraftBukkit! (1.9 through 1.14.*)
 - ProtocolLib (4.0 or better)
 
-### License:
+### Configurate
+1. Download [ProtocolLib](https://www.spigotmc.org/resources/protocollib.1997)
+2. Download [Orebfuscator](https://github.com/Imprex-Development/Orebfuscator/releases/latest)
+3. Restart your minecraft server
+4. That was it.
+
+## Developer
+
+### API
+```java
+Bukkit.getServicesManager().getRegistration(Orebfuscator.class).getProvider();
+```
+or use
+```java
+Orebfuscator.getProvider();
+```
+### Maven
+
+### Clone
+1. Clone this repo "git clone https://github.com/Imprex-Development/Orebfuscator.git"
+2. Open eclipse and right click on the "Project Explorer"
+3. Click "Import..."
+4. Maven -> Existing Maven Projects
+5. Select the downloaded repo
+6. Finished
+
+## Build
+1. Click right click on the Orebfuscator-repo folder and select "Run as" -> "Maven Build..."
+2. Put into Goals this "clean compile package -pl Plugin -Dorebfuscator-version=5.0.0 --also-make"
+3. Click Run
+4. Your jar will be builded under the folder "target"
+5. Finished
+
+## Release a new version
+1. git tag **version**
+2. git push origin **version**
+3. Finished
+
+## License:
```

Comment:
Just leaving a note here how utterly impressed I am at the respect right here -- thanks for leaving the license and dev notes "in place" -- I see on many handovers eager devs erase the past and it's always sad to see, but not here and I thank you for that.

Reply:
Thanks 😄

We hope that you like the new version [Imprex](https://github.com/Imprex-Development/Orebfuscator)
currently in development